mage_ai/server/app.py

```diff
@@ -279,9 +280,15 @@ def update_pipeline(id):
     if feature_set_id is not None:
         feature_set = FeatureSet(id=feature_set_id)
         df_transformed = clean_pipeline.transform(feature_set.data_orig, auto=False)
+
+        # TODO: Come up with better way to support dynamic rulesets for pipelines
+        rules = [rule for rule in DEFAULT_RULES]
+        if clean_pipeline.outliers_removed:
+            rules.remove(RemoveOutliers)
```

Comment:
we don't need this logic here now since we don't include multidimensional outlier removal in suggestions.